src/client/cli/formatter/table_formatter.cpp

```diff
@@ -123,11 +123,19 @@ std::string mp::TableFormatter::format(const ListReply& reply) const
     if (reply.instances().empty())
         return "No instances found.\n";
 
-    fmt::format_to(buf, "{:<24}{:<18}{:<17}{:<}\n", "Name", "State", "IPv4", "Image");
+    size_t name_column_width = 24;
+    for (const auto& instance : reply.instances())
+    {
+        auto instance_name_len = instance.name().length();
+        if (instance_name_len >= name_column_width)
+            name_column_width = instance_name_len + 1;
```

Comment:
Tiny detail, but why not add 1 to the maximum length? I mean doing `+ 1` outside of the loop, to keep it focused on finding the max and accounting for the space separately

Reply:
variable name is "name_column_width" so I have to store column width :)
if I rename variable to "max_instance_name_len" then I don't need to +1.

Reply:
I see what you mean. Since it is not a constant and its ultimate value takes a few lines to calculate anyway, I would not see the naming as wrong necessarily (it is not really the column width either until it has gone through the instruction that finds the largest name). But the intermediate var you ended up using is perhaps the cleaner approach.